hyundai_kia_connect_api/HyundaiBlueLinkAPIUSA.py

```diff
@@ -44,7 +44,9 @@ def __init__(
         self,
         region: int,
         brand: int,
+        language: str
     ):
+        self.LANGUAGE: str = language
```

Comment:
Since this isn't API specific but rather session I would think token would be a better place for it.